cellpack/autopack/writers/__init__.py

```diff
@@ -175,7 +175,10 @@ def save_as_simularium(self, env, all_ingr_as_array, compartments):
         file_name = env.helper.writeToFile(
             env.result_file, env.boundingBox, env.name, env.version
         )
-        autopack.helper.post_and_open_file(file_name)
+        if env.config_data is None or env.config_data.get(
+            "upload_results", env.config_data.get("number_of_packings", 1) <= 1
+        ):
```

Comment:
two things: I don't thin env.config_data can ever be none, so I don't think you need that check
I think you could do 3 or less. I have run some multiple packings recently that I would like to see the results of.